flixel/text/FlxText.hx

```diff
@@ -59,6 +59,15 @@ class FlxText extends FlxSprite
 	 */
 	public var size(get, set):Int;
 
+	/**
+	 * A number representing the amount of space that is uniformly distributed
+	 * between all characters. The value specifies the number of pixels that are
+	 * added to the advance after each character.
+	 *
+	 * The default value is `null`, which means that 0 pixels of letter spacing is used.
+	 */
+	public var letterSpacing(default, set):Null<Float> = null;
```

Comment:
1. (default, set) will cause issues, since set_ letterSpacing only sets _defaultFormat.letterSpacing. meaning the following will happen:
```hx
text.letterSpacing = 1;
trace(text.letterSpacing); // null
```
We should add a getter that returns `_defaultFormat.letterSpacing`

looks like the other fields avoid using Null<T> and just use primitive types, I think Null<T> should be avoided if possible. let's use `Float` and set the default value to 0, and also set the _defaultFormats initial letterSpacing to 0

Reply:
The letterSpace is null by default in openfl

Reply:
That value cannot be getted

Reply:
https://github.com/openfl/openfl/blob/6c106bc8463edb1ee71491d1e4f2b7e1dbf15f7f/src/openfl/text/TextFormat.hx#L156

Reply:
> That value cannot be getted

Sure it can

> The letterSpace is null by default in openfl
I understand, that's why i said "set the _defaultFormats initial letterSpacing to 0", specifically, [here](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/text/FlxText.hx#L221)